### PR TITLE
New new tic tac

### PR DESCRIPTION
**Pull request by Java-Dev-Art**

### CODE REVIEW

**Review by svetamolchanova — COMMENTED** [review]
Please, leave only class with Tictactoe, other will be deleted. They were checked in other pull requests. Did you create new branch from current? We create new branch from master

**Comment by svetamolchanova** [comment]
Leave all files in branch. I will check only new. But create new branches from master in future

File: src/HomeWork5/Tictactoe.java [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                break;
+            }
+            gameX ++;
+
+        }
+    }
+
+
+
+
```
Comment (svetamolchanova) [inline src/HomeWork5/Tictactoe.java]: delete unnecessary spaces

Comment (Java-Dev-Art) [reply src/HomeWork5/Tictactoe.java]: I fixed tictactoe in this branch. Can you check

Comment (Java-Dev-Art) [reply src/HomeWork5/Tictactoe.java]: commit bugFix

**Pull request closed without merging**